### DISCUSSION TITLE
How to mount a MooseFS with Dokany on Windows?

### DISCUSSION THREAD

**Discussion by hwhsu1231** [moosefs/moosefs#530] [Q&A]
### Problem Descrption

I asked this question in Dokany: https://github.com/dokan-dev/dokany/issues/1135

And they told me to ask MooseFS directly.

I wondered how do I use Dokany to mount a MooseFS on Windows?

### Screenshots

![image](https://user-images.githubusercontent.com/71438617/224697157-fe238dd5-1040-48e2-8732-ab22f0287390.png)

**Comment by chogata** [comment]
MooseFS client for Windows is a paid feature. If you are interested, contact us at contact@moosefs.com
